src/libcommon/comm.h

```diff
@@ -109,6 +109,8 @@ typedef enum {
     REQUEST_NUM_UTILITY_CHECKCOMMSTATUS,
     REQUEST_NUM_UTILITY_HASSYSTEMLAUNCHONSTARTUP,
     REQUEST_NUM_UTILITY_HASLAUNCHONSTARTUP,
+    REQUEST_NUM_UTILITY_SET_KEYVALUE,
+    REQUEST_NUM_UTILITY_GET_KEYVALUE,
     REQUEST_NUM_UTILITY_SETLAUNCHONSTARTUP,
```

Comment:
To move after REQUEST_NUM_UTILITY_HASLAUNCHONSTARTUP

Reply:
I movedit in commit: https://github.com/Infomaniak/desktop-kDrive/pull/76/commits/0a3ea500340aec701b225bf3ee1c689a645865f5